content/prerequisites/k8stools.md

```diff
@@ -20,7 +20,9 @@ mkdir -p ~/.kube
 
 #### Install kubectl
 ```
-sudo curl --silent --location -o /usr/local/bin/kubectl "https://amazon-eks.s3-us-west-2.amazonaws.com/1.11.5/2018-12-06/bin/linux/amd64/kubectl"
+curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.14.0/bin/linux/amd64/kubectl
```

Comment:
I don't mind swapping to the official release, but can we keep to the current eks version? (currently 1.12.7)... 

also, using long options, and curling directly into the target location saves us some explain time (when in person). Can we stick to the command format, and just change the url?